app/views/catalog/_show_collection.html.erb

```diff
@@ -3,6 +3,14 @@
     <%= render partial: 'show_sidebar' %>
   </div>
   <div class='col-md-9'>
+    <div class="btn-toolbar" role="toolbar" aria-label="Toolbar with button groups">
```

Comment:
Do you mind pulling these buttons out? I think they will probably be added as part of #149